src/js/apps/clinicians/clinicians-all_app.js

```diff
@@ -93,14 +93,14 @@ export default SubRouterApp.extend({
       onSubmit: () => {
         clinicianModal.disableSubmit();
         clinician.saveAll(clinicianClone.attributes)
-          .then(({ data }) => {
+          .then(() => {
```

Comment:
all backbone requests now return the model instead of the data, which we already have above.